Documentation/sources/exd-cbe-beta/testapp-setup.md

```diff
@@ -0,0 +1,23 @@
+# Test app setup
+
+## Setup
+
+Please follow the below setup steps for iOS apps:
+
+1. Install Xcode - https://developer.apple.com/xcode/
+2. Get the Messaging SwiftUI test app from the GitHub repository - https://github.com/adobe/aepsdk-messaging-ios by following option 2a or 2b.
+    a. Clone the code from https://github.com/adobe/aepsdk-messaging-ios.git. Checkout `exd-cbe-beta` branch.
+    ![Clone the repository](../../assets/clone-the-repo.png)
+    b. Select the `exd-cbe-beta` branch and download the code ZIP file.
```

Comment:
need to work on some of the markdown formatting here to make it look a little cleaner in the browser